cmsplugin_feed_ai/models.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from cms.models import CMSPlugin
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = (
+    "ProfileFeedPluginConf",
+)
+
+
+class ProfileFeedPluginConf(CMSPlugin):
+    """
+    Plugin model for storing profile feed
+    related configuration.
+    """
+    items_per_service = models.PositiveSmallIntegerField(
+        default=5,
+        verbose_name=_("number of items per service"),
+        help_text=_(
+            "Determines the max. number of items to be shown per social "
```

Comment:
Come to think of it, I don't think this help text really provides any extra information c.f. the verbose name.